configurations/ldap-authenticator.yaml

```diff
@@ -0,0 +1,5 @@
+authenticator: 'com.scylladb.auth.SaslauthdAuthenticator'
```

Comment:
since we already have configuration that is "ldpa-authorization".
We should have one more file (in addition to this one you're adding which is: ldap-authorization-and-authentication.yaml)

Reply:
Ok, will add a new yaml